models/staging/tpdm/base/base_tpdm__candidate_educator_preparation_program_associations.sql

```diff
@@ -0,0 +1,35 @@
+with candidate_educator_preparation_program_associations as (
+    {{ source_edfi3('candidate_educator_preparation_program_associations') }}
+),
+renamed as (
+    select 
+        tenant_code,
+        api_year,
+        pull_timestamp,
+        last_modified_timestamp,
+        file_row_number,
+        filename,
+        is_deleted,
```

Comment:
would it make sense to define the type for these attributes as well?
For example `is_deleted` should be a boolean.

/cc @holdorph thoughts?